app/controllers/providers/proceedings_sca/interrupts_controller.rb

```diff
@@ -5,6 +5,12 @@ def show
         type
       end
 
+      def destroy
+        legal_aid_application.proceedings.order(:created_at).last.destroy!
```

Comment:
order_by(:created_at).last feels a bit fragile, is there no way we can pass the proceeding to the page and be sure?